src/components/recipeList.js

```diff
@@ -11,14 +11,21 @@ class recipeList extends React.Component {
   }
 
   render() {
+    const { recipes } = this.props;
+
     return (
       <div className="recipe-list">
         <h2 className="recipe-list-head">Showing 'chicken' recipes</h2>
-
         <div className="recipe-list-cards">
-          <Recipe />
-          <Recipe />
-          <Recipe />
+          {recipes.map(recipe => (
```

Comment:
Ahh I was having such a hard time figuring out where to map through the recipes, but this makes sense!